m4/trilinos.m4

```diff
@@ -197,9 +201,11 @@ AC_DEFUN([CONFIGURE_TRILINOS_10],
        dnl header files whose absence indicates Epetra is not available,
        dnl we just choose one here that libmesh actually includes.
        dnl ------------------------------------------------------
-       AC_CHECK_HEADER([$withtrilinosdir/include/Epetra_CombineMode.h],
```

Comment:
Yes - the Epetra_config.h should guarantee the existence of Epetra_CombineMode.h. The Epetra_config.h is generated during configuration so it is actually a more strict test since it makes sure that trilinos has been configured correctly as well.